src/scss/components/_progress-indicator.scss

```diff
@@ -11,79 +9,148 @@
   }
 }
 
-.#{prefix('progress-indicator')} {
-  @extend .openforms-card; // TODO -> syntax highlighting trips on #{prefix('card')}
-
+/**
+ * Desktop and mobile styling for the progress indicator element.
+ *
+ * The parent component (_app.scss) is responsible for re-arranging the order of
+ * elements depending on the viewport. The styles of this component are responsible for
+ * managing how it looks/behaves on mobile/non-mobile viewports:
+ *
+ * - on mobile, the element takes a navbar-like approach that can be expanded/collapsed
+ * - on desktop, it behaves like a sticky element, so that it can be placed in a sidebar
+ *
+ * TODO: remove the fallbacks/defaults after a deprecation period. Many of these
+ * design tokens here didn't exist before 2.1 and had hardcoded values.
+ */
+.openforms-progress-indicator {
+  @include card.card('progress-indicator');
+
+  position: sticky;
+  inset-block-start: var(
+    --of-progress-indicator-inset-block-start,
+    var(--of-app-grid-column-gap, 20px)
+  );
+
+  // Do not display the toggle button on non-mobile devices
   @include bem.element('mobile-header') {
-    @include body;
-    @include body--big;
-    @include margin(-$grid-container-margin, $properties: margin-left);
-
-    // style for replacing div with a button for accessibility
-    background: none;
-    border: none;
-    padding: 0;
-    cursor: pointer;
-    width: calc(100% + $grid-container-margin);
-
-    @include show-on-mobile(flex);
-
-    align-items: center;
-
-    .fa-icon {
-      display: block;
-      flex-shrink: 0;
-      flex-basis: $grid-container-margin * 2;
-      text-align: center;
-    }
+    display: none;
   }
 
-  @include bem.element('form-title') {
-    @include ellipsis;
-    font-weight: bold;
+  @include bem.element('nav') {
+    // box-sizing: border-box;
```

Comment:
I don't see a difference if I enable it, should we remove it?

Reply:
ah left-over experiment, I will remove it indeed!